importer/SeBbrSv.py

```diff
@@ -28,16 +28,52 @@ def update_labels(self):
             self.add_to_report("malformed_label", self.namn)
         self.add_label("sv", label)
 
-    def set_bbr(self):
+    def create_bbr_link(self):
         """
-        Set the BBR ID property.
+        Create BBR link.
 
         This will get a link that looks like
             raa/bbr/21300000002805
         Depending on whether the prefix is raa/bbr/ or raa/bbra/
         """
-        bbr_link = utils.get_bbr_link(self.bbr)
-        self.add_statement("cultural_heritage_sweden", bbr_link)
+        self.bbr_link = utils.get_bbr_link(self.bbr)
+
+    def set_bbr(self):
+        """Set the BBR ID property."""
+        self.add_statement("cultural_heritage_sweden", self.bbr_link)
+
+    def get_online_data(self):
+        """
+        Retrieve online data about BBR complex.
+
+        This includes, inter alia, the legal protection
+        status and ID's of each part of the
+        compound.
+        """
+        h_property = self.props["cultural_heritage_sweden"]
+        kulturarv_id = self.wd_item["statements"][h_property][0]["value"]
+        url = "http://kulturarvsdata.se/{}".format(kulturarv_id)
```

Comment:
could you possibly store the id as `self.kulturarv_id` since you recalculate it once on L129 and once on L138?